wsmaster/che-core-sql-schema/src/main/resources/che-schema/6.0.0/9__increase_externalmachine_env_value_length.sql

```diff
@@ -0,0 +1,12 @@
+--
+-- Copyright (c) 2012-2017 Red Hat, Inc.
+-- All rights reserved. This program and the accompanying materials
+-- are made available under the terms of the Eclipse Public License v1.0
+-- which accompanies this distribution, and is available at
+-- http://www.eclipse.org/legal/epl-v10.html
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+ALTER TABLE externalmachine_env ALTER COLUMN env_value TYPE TEXT;
```

Comment:
Please, add notion about this into the PR description.

Reply:
done